<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addSpentMonth</class>
 <widget class="QDialog" name="addSpentMonth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar novo mês de gastos</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Mês:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ano:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_month">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <item>
        <property name="text">
         <string>Janeiro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fevereiro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Março</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Abril</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Maio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Junho</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Agosto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Setembro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Outubro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Novembro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dezembro</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="comboBox_year">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string>Adicionar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
